acme.sh

```diff
@@ -1999,7 +1999,13 @@ _post() {
     if [ "$_ret" != "0" ]; then
       _err "Please refer to https://www.gnu.org/software/wget/manual/html_node/Exit-Status.html for error code: $_ret"
     fi
-    _sed_i "s/^ *//g" "$HTTP_HEADER"
+    if echo $_WGET | grep -q " -d " 2> /dev/null; then
+      # Demultiplex wget debug output
+      cat "$HTTP_HEADER" >&2
+      _sed_i '/^[^[:space:]][^[:space:]]/d; /^[[:space:]]*$/d' "$HTTP_HEADER"
+    fi
+    # remove leading whitespaces from header to match curl format
+    _sed_i 's/^[[:space:]][[:space:]]//g' "$HTTP_HEADER"
```

Comment:
don't use `[[:space:]]`,  just use the literal space ` `

Reply:
I tried to stay consistent with the rest of the code in acme.sh.
Am I missing something?

Reply:
never mind the other dns api.

Reply:
ok